cli_tests/app.sh

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env bats
+
+# To run locally:
+# export KETCH_COMMAND=<location of ketch binary>
+# assure you have a kubernetes cluster running w/ traefik, cert manager, etc. (see ketch getting started docs)
+# assure the ketch cli is compiled (make ketch)
+# assure you have bats installed locally (via apt, brew, etc.)
+# ./cli_tests/app.sh
+
+setup() {
+    if [[ -z "${KETCH_COMMAND}" ]]; then
+    KETCH=$(pwd)/bin/ketch
+  else
+    KETCH="${KETCH_COMMAND}"
+  fi
+  INGRESS=$(kubectl get svc traefik -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+  FRAMEWORK="myframework"
+  APP_IMAGE="docker.io/shipasoftware/bulletinboard:1.0"
```

Comment:
could you build a lite-weight image for testing purposes instead of this, especially for tests? Runners usually have very low CPU and memory, any such improvement would be nice! :)

Reply:
I sure can. I pushed a super-quick image to our gcp CI GCR repo for testing. I'm hoping that is what you're after.